tcpdf.php

```diff
@@ -23939,6 +23939,12 @@ protected function removeTagNamespace($name) {
 	 */
 	protected function startSVGElementHandler($parser, $name, $attribs, $ctm=array()) {
 		$name = $this->removeTagNamespace($name);
+        if (array_key_exists('xlink:href', $attribs)) {
+            if (!array_key_exists('href', $attribs)) {
+                $attribs['href'] = $attribs['xlink:href'];
+            }
+            unset($attribs['xlink:href']);
+        }
```

Comment:
What if you invert this part ?
It would avoid having to rename the property on the lines below ?